training/finetune_acft_hf_nlp/src/model_selector/model_selector.py

```diff
@@ -82,6 +82,12 @@ def model_selector(args: Namespace):
     task_runner = get_task_runner(task_name=args.task_name)()
     task_runner.run_modelselector(**vars(args))
 
+    # additional logging
+    if hasattr(args, "model_name"):
```

Comment:
Do you think adding `getattr` in printing the value adds any more value - `logger.info(f"Model name: {getattr(args, "model_name", None)}`? This will log the value even if there is no attribute in the args.

Reply:
updated